projects/BFDR/BirthRecord_submissionProperties.cs

```diff
@@ -3140,5 +3140,104 @@ public string FatherEthnicityLiteral
 
             }
         }
+
+        private int? GetCigarettesSmoked(string code)
+        {
+            var entry = Bundle.Entry.Where(e => e.Resource is Observation obs && CodeableConceptToDict(obs.Code)["code"] == code).FirstOrDefault();
+            if (entry != null)
+            {
+                Observation observation = (Observation)entry.Resource;
+                return (observation.Value as Hl7.Fhir.Model.Integer)?.Value;
+            }
+            return null;
+        }
+
+        private void SetCigarettesSmoked(string code, int? value)
+        {
+            var entry = Bundle.Entry.Where(e => e.Resource is Observation obs && CodeableConceptToDict(obs.Code)["code"] == code).FirstOrDefault();
+            if (entry == null)
+            {
+                Observation obs = new Observation();
+                obs.Id = Guid.NewGuid().ToString();
+                obs.Code = new CodeableConcept(VR.CodeSystems.LOINC, code);
+                obs.Value = new Hl7.Fhir.Model.Integer(value);
+                obs.Subject = new ResourceReference($"urn:uuid:{Child.Id}");
```

Comment:
Looking at the IG, should Subject be Mother and just the Focus is Child? https://build.fhir.org/ig/HL7/fhir-bfdr/StructureDefinition-Observation-cigarette-smoking-before-during-pregnancy.html

Reply:
Good catch, will fix.